batch/batch/worker/worker.py

```diff
@@ -2273,8 +2274,11 @@ async def run(self):
 
                 self.state = 'running'
 
-                with self.step('downloading_jar'):
-                    local_jar_location = await self.download_jar()
+                if os.environ.get('HAIL_TERRA'):
+                    local_jar_location = '/hail-jars/hail-all-spark.jar'
+                else:
+                    with self.step('downloading_jar'):
+                        local_jar_location = await self.download_jar()
```

Comment:
this also feels like a worker API level thing. Like `get_local_jar()` or w/e.

Reply:
Ehh. bleh, it depend on the worker's jar locks and the pool. You could pass that in I guess but it does start to feel a bit weird that we moved all that functionality to a different class. Let's keep it as is.